### ISSUE TITLE
"Folders" do nothing in DSub with Browse By Tags

### ISSUE THREAD

**Issue by apiontek** [airsonic/airsonic#616]
## Problem description

I have three "folders" in Airsonic: Music, Audiobooks, and Podcasts. In the web interface for both Airsonic and Subsonic, if I pick one of the "folders" it shows me only content from those folders, as expected.

However, in DSub (v5.3.5) on Android, while Subsonic (6.1.3) yields the same behavior, Airsonic does not.

In my DSub server settings, with "Browse By Tags" disabled, Airsonic behaves appropriately. Selecting a "folder" shows only the relevant items (podcasts in Podcasts, audiobooks in Audiobooks, etc.)

But I far prefer "Browse By Tags," and with that enabled, Airsonic behaves strangely. Selecting a "folder" shows my entire library. No matter what I choose, I see content from all three "folders."

Subsonic displays only the content from a selected folder, whether Browse By Tags is enabled or not, which I think is expected behavior.

### Steps to reproduce

1. Run Airsonic with more than one "folder" of recognizably different content.
2. Use DSub on Android with said Airsonic server and try browsing the different shared folders with Browse By Tags enabled

## System information

 * **Airsonic version**: 10.1.0-RELEASE
 * **Operating system**: Ubuntu 16.04, inside an LXD container on Ubuntu 17.04
 * **Proxy server**: nginx/1.12.1
 * **Client**: DSub 5.3.5

**Comment by CharlesSchaack** [comment]
I know you said this isn't exactly how subsonic works but wouldn't this be what is expected? You are browsing by tags. Once you turn that setting on it should only display albums by their tags and all folders should essentially be ignored.

I assume you are wanting to keep the folders separate to act like different 'libraries' but yet within those folders you are wanting to browse by tag?

**Comment by apiontek** [comment]
Yes, I want to keep the top-level "media folders" separate, as libraries, but browse by tags within those folders. There's no other way to have different media types and keep them separate. All folders within a top-level media source directory should be ignored, with browse by tag -- that makes sense. But I think the top-level media source directories should still be reflected.

That's how Subsonic does it. I think it's an API change as of API version 1.14.0, according to the supysonic dev (I opened [a similar issue there](https://github.com/spl0k/supysonic/issues/74)).

Alternatively as airsonic develops on its own trajectory, if a "Library" concept were to be introduced, that'd be fine (i.e., Plex distinguishes between libraries & media source directories), but of course then that'd affect compatibility with Subsonic apps like DSub et al.
